src/components/Modals/PublishWizardCollectionModal/PayPublicationFeeStep/PayPublicationFeeStep.tsx

```diff
@@ -139,14 +147,15 @@ export const PayPublicationFeeStep: React.FC<
     <Modal.Content>
       <Column>
         <Row className={styles.details}>
-          {isLoading && (
-            <div className={styles.loadingOverlay}>
-              <Loader inline size="massive" />
-              {publishingStatus === AuthorizationStepStatus.PROCESSING
-                ? 'Submitting for review'
-                : t('publish_wizard_collection_modal.accept_in_wallet')}
-            </div>
-          )}
+          {isLoading ||
+            (isLoadingAuthorization && (
+              <div className={styles.loadingOverlay}>
+                <Loader inline size="massive" />
+                {publishingStatus === AuthorizationStepStatus.PROCESSING || (isUsingMagic && isMagicAutoSignEnabled)
+                  ? 'Submitting for review'
```

Comment:
do you mind adding this string in the i18n files?

Reply:
Done!